internal/pkg/dl/output_health.go

```diff
@@ -0,0 +1,54 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package dl
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/elastic/fleet-server/v7/internal/pkg/bulk"
+	"github.com/gofrs/uuid"
+)
+
+type DataStream struct {
+	Dataset   string `json:"dataset,omitempty"`
+	Type      string `json:"type,omitempty"`
+	Namespace string `json:"namespace,omitempty"`
+}
+
+type OutputHealth struct {
+	Output     string     `json:"output,omitempty"`
+	State      string     `json:"state,omitempty"`
+	Message    string     `json:"message,omitempty"`
+	Timestamp  string     `json:"@timestamp,omitempty"`
+	DataStream DataStream `json:"data_stream,omitempty"`
+}
```

Comment:
If this is something that is being written to ES, does it make more sense to define in in model/schema.json instead?

Reply:
moved to schema.json